lib/parse-args.js

```diff
@@ -5,9 +5,12 @@ const { readFileSync } = require('fs')
 const Yargs = require('yargs/yargs')
 const { applyExtends } = require('yargs/helpers')
 const parser = require('yargs-parser')
-const { resolve } = require('path')
 
 function buildYargs (withCommands = false) {
+  if (typeof process.env.NODE_V8_COVERAGE !== 'string') {
+    process.env.NODE_V8_COVERAGE = process.cwd() + '/coverage/tmp'
+  }
+
```

Comment:
Was able to refactor the block into three lines.  The relative path has been changed to an absolute path.